.github/workflows/code_style.yml

```diff
@@ -3,14 +3,14 @@ on: [push, pull_request]
 
 jobs:
   nGraph:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
`clang-format-9` is available on 18.04. Why we need this change?

Reply:
Maybe it's not needed, but why not?

Reply:
No objection. Just curiosity.